jenkins/ignoreArchiveNotReadableTest.patch

```diff
@@ -0,0 +1,22 @@
+# Disabling single test which fails.  Manually checked that Java does set the read only file permissions correctly
+# Requires more investigation, early thoughts could be related to the jenkins test harness coming from maven central and the test not using the one built from source
```

Comment:
Running this as `root` would do this.

Reply:
Ah that explains it - many thanks @daniel-beck I was definitely scratching my head on that one :) Out of all the tests in Jenkins this was the only one that failed.

@kaniini do we have any precedence for running steps as different users?

Reply:
Just testing using `su-exec` and a different user, looks promising.

Reply:
Not really, we create a throwaway container which is unprivileged on the outside, so I haven't given it much thought.

Reply:
Ok thanks - makes sense.  We might need to revisit this and in the meantime keep the patch to ignore that particular test.